src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -323,42 +327,27 @@ const MarkerHeatmap = ({ experimentId }) => {
   const renderExtraPanels = () => (
     <>
       <Panel header='Gene selection' key='gene-selection'>
-        <Tabs defaultActiveKey='1'>
-          <TabPane tab='Add/Remove genes' key='1'>
-            <MarkerGeneSelection
-              config={config}
-              onUpdate={updatePlotWithChanges}
-              onGeneEnter={onGeneEnter}
-              onReset={onReset}
-            />
-            <div>
-              <p>Gene labels:</p>
-              <Radio.Group
-                onChange={
-                  (e) => updatePlotWithChanges({ showGeneLabels: e.target.value })
-                }
-                value={config.showGeneLabels}
-              >
-                <Radio value>Show</Radio>
-                <Radio value={false}>Hide</Radio>
-              </Radio.Group>
-            </div>
-          </TabPane>
-          <TabPane tab='Search for and re-order genes' key='2'>
-            <p>Type in a gene name and select it to add it to the heatmap. Drag and drop genes to re-order them.</p>
-            {/* space needed to separate search box and reorder tree, display=flex fills the space */}
-            <Space direction='vertical' style={{ display: 'flex' }}>
-              <GeneSearchBar
-                plotUuid={plotUuid}
-                experimentId={experimentId}
-                searchBarUuid={searchBarUuid}
-              />
-              <GeneReorderTool
-                plotUuid={plotUuid}
-              />
-            </Space>
-          </TabPane>
-        </Tabs>
+        <MarkerGeneSelection
+          config={config}
+          plotUuid={plotUuid}
+          searchBarUuid={searchBarUuid}
+          experimentId={experimentId}
+          onUpdate={updatePlotWithChanges}
+          onReset={onReset}
+          onDataChange={onDataChange}
+        />
+        <div style={{ paddingTop: '10px' }}>
```

Comment:
Instead of using this `<div>`, does replacing it with `<Space>` give the same spacing effect?

Reply:
A div with padding is needed to create some distance between the reset button and the `Gene labels:` text. Replacing it with a space would instead space the text and the buttons below it.